xen.spec

```diff
@@ -781,6 +781,7 @@ rm -rf %{buildroot}
 %if %build_efi
 /boot/efi/EFI/qubes/*.efi
 %endif
+/usr/lib/debug/xen*
```

Comment:
This doesn't look like appropriate place for debug symbols. It should be part of debuginfo package.

Reply:
I followed: http://pkgs.fedoraproject.org/cgit/rpms/xen.git/tree/xen.spec#n875 is this not good?

Reply:
Well, it should be in debuginfo... But if upstream have no idea how to do that either, then lets keep it here for now.
BTW the actual upstream commit putting those files in hypervisor package is this one: http://pkgs.fedoraproject.org/cgit/rpms/xen.git/commit/?id=ee3a55519e2472218240ad6096334c9aa45d887d